launcher/ui/widgets/ModFilterWidget.cpp

```diff
@@ -115,6 +117,13 @@ ModFilterWidget::ModFilterWidget(MinecraftInstance* instance, bool extended, QWi
     prepareBasicFilter();
 }
 
+bool ModFilterWidget::eventFilter(QObject* obj, QEvent* evt)
+{
+    if ((obj != ui->versions && obj != ui->version) || evt->type() != QEvent::Wheel)
+        return QTabWidget::eventFilter(obj, evt);
+    return true;
+}
```

Comment:
Couldn't this be in CheckComboBox::eventFilter

Reply:
will still need this function for `ui->version` as that one is the normal checkbox.
I can move a part of it but I will still need to keep the eventFilter on `ModFilterWidget` just for `ui->version`, are you ok with that?

Reply:
ok moved that and also implemented the paint event to make it look like a normal QComboBox.